testpackage/small_test_ukko_gpu_github_ci.sh

```diff
@@ -0,0 +1,428 @@
+#!/bin/bash
+#SBATCH -t 01:30:00        # Run time (hh:mm:ss)
+#SBATCH --job-name=CI_gpu
+#SBATCH -M ukko
+# test short medium 20min1d 3d
+#SBATCH -p gpu
+#SBATCH --mem-per-gpu=20G
+#SBATCH --cpus-per-gpu=8
```

Comment:
could add:
#SBATCH --constraint=a100

And either add multithreading hint or, perhaps better, turn off hyperthreading in this job

Reply:
Alright, done!